molgenis/molgenis-emx2
# feat(central): Show progress when creating a new schema from a template

Closes [#3059](https://github.com/molgenis/molgenis-emx2/issues/3059)
What are the main changes you did:
- Wrapped the graphql create mutation in a task so that the progress can be async tracked in the front end

how to test:
- Go to https://preview-emx2-pr-4123.dev.molgenis.org/apps/central/#/
- Create a new schema with any profile (e.g. Beacon_V2) and see the progress


todo:
- [x] Create a ImportTask (and TableStore?) for profile imports
- [x] Create a ImportTask for non profile data models
- [x] Integrate with GraphQL API
- [x] Update the front-end to show the task
